package.json

```
@@ -26,12 +29,17 @@
     "@nestjs/jwt": "^10.2.0",
     "@nestjs/passport": "^10.0.3",
     "@nestjs/platform-express": "^10.0.0",
+    "@nestjs/swagger": "^7.3.1",
     "@nestjs/typeorm": "^10.0.2",
     "bcrypt": "^5.1.1",
+    "class-validator": "^0.14.1",
     "cookie-parser": "^1.4.6",
     "dotenv": "^16.4.5",
+    "mysql": "^2.18.1",
+    "passport-jwt": "^4.0.1",
     "reflect-metadata": "^0.2.0",
-    "rxjs": "^7.8.1"
+    "rxjs": "^7.8.1",
+    "swagger-ui-express": "^5.0.0"
```

```suggestion
    "swagger-ui-express": "^5.0.0",
    "typeorm": "^0.3.20"
```
TypeORM에 대한 Node Dependencies가 추가되어 있지 않습니다.
일단 제 프로젝트에서 추가했는데 로컬에서 설치된 버전 확인 후 package에 추가해주세요.
passport도 없어요.

---

수정하였습니다.

---

수정했습니다.